exercise_05/isAdditionGreaterThanFifty.js

```diff
@@ -4,3 +4,16 @@
  * input: number (summand2)
  * output: boolean
  */
+
+/**
+ * Check if the sum of two numbers is 50 or greater.
+ * @param {number} summand1
+ * @param {number} summand2
+ * @returns {boolean}
+ */
+
+const isAdditionGreaterThanFifty = function(summand1, summand2) {
+    return (summand1 + summand2) >= 50;
```

Comment:
Avoid magic numbers. 

```suggestion
    return (summand1 + summand2) >= THRESHOLD;
```

- [ ] Extract number to constant